gensim/corpora/wikicorpus.py

```diff
@@ -546,7 +528,11 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
         token_max_len : int, optional
             Maximal token length.
         lower : bool, optional
-             Convert all text to lower case?
+             Convert all text to lower case.
```

Comment:
-1: should be a question (here and elsewhere).

Reply:
fixed